paper.md

```diff
@@ -50,28 +50,30 @@ Intracranial electrophysiology analysis requires precise location and anatomical
 
 For brain electrophysiology research using stereoelectroencephalography (sEEG) and electrocorticography (ECoG), the raw electrophysiology, neurosurgical planning and neuroimaging needs to be integrated before meaningful interpretation of signal processing analyses can occur. Typically, for a patient with an intracranial electrophysiology recording, two anatomical images are collected. A preoperative magnetic resonance (MR) image is collected to obtain detailed individual brain anatomy, and a postoperative computed tomography (CT) image is collected to get a high-resolution image with 3D contact locations (although this can also be provided by a postoperative MR). The surgical plans then link the names of the contacts in the recording with their positions.
 
-To integrate these electrophysiology and imaging data, first, the CT must be aligned to the MR in order to determine the positions of the electrode contacts in the CT relative to brain structures in the MR. In MNE-Python, these are aligned using a mutual information histogram-matching approach using DIPY [@GaryfallidisEtAl2014] that works without manual pre-alignment.
+To integrate these electrophysiology and imaging data, first, the CT must be aligned to the MR in order to determine the positions of the electrode contacts in the CT relative to brain structures in the MR. In MNE-Python, these are aligned using a mutual information histogram-matching approach using DIPY [@GaryfallidisEtAl2014] that works without manual pre-alignment using the function ``mne.transforms.apply_volume_registration``.
```

Comment:
why double backticks? single should be enough for markdown

Reply:
Ah dang, that's how I had it originally but I thought it was wrong so I changed it, I'll revert